src/main/java/frc/robot/subsystems/conveyor/commands/CargoIn.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Ports;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class CargoIn extends CommandBase {
```

Comment:
"Intake" is a better name.

Reply:
I don't think so, we would easily confuse with the intake subsystem

Reply:
> I don't think so, we would easily confuse with the intake subsystem

"cargo in" can be confused with the intake too.